frontend/src/pages/CompareV2.tsx

```diff
@@ -313,10 +375,20 @@ function getDropdownItems(filteredRunArtifacts: RunArtifact[]) {
   return dropdownItems;
 }
 
-function getArtifactFromSelectedItem(
+const getNamespace = (selectedItem: SelectedItem, filteredRunArtifacts: RunArtifact[]) => {
+  const selectedRun = filteredRunArtifacts.find(
+    runArtifact => runArtifact.run.run?.name === selectedItem.itemName,
+  )?.run;
+  if (selectedRun) {
+    const jsonWorkflow = JSON.parse(selectedRun.pipeline_runtime!.workflow_manifest || '{}');
```

Comment:
`workflow_manifest` is argo workflow which is V1 specific. (It is still available for now but we should avoid using it in V2 features. Reference: https://www.kubeflow.org/docs/components/pipelines/reference/api/kubeflow-pipeline-api-spec/#/definitions/apiPipelineRuntime)

Can you ask this question in KFP working group about `How do I get k8s namespace information about a pipeline run without relying on argoworkflow`? And let's also add a TODO here to move away from workflow_manifest.

Reply:
Sounds good - I've added a TODO (https://github.com/kubeflow/pipelines/pull/7981/commits/385714c10c7d013f77ddaef7a49e8c283bf7d2f7) and asked that question (with some additional context) in the KFP working group. Thank you!

Reply:
Awesome, thank you Chris!